layers/+readers/pdf/README.org

```diff
@@ -49,6 +49,24 @@ If you use a Debian based system such as Ubuntu, you can skip this step, as
 =pdf-tools= installation asks whether you want it to try to download
 automatically all dependencies using =apt-get=.
 
+On macOS, the default instructions might not work. There is a brew tap at dunn/emacs that contains pdf-tools, but it did not work for me. I used the following successfully. Note that your PATH should be set correctly in Emacs, and that pdf-tools does not work in terminal (GUI-only).
+
+#+BEGIN_SRC
```

Comment:
Org source blocks must always set a language. See CI error:
`error: Language not specified in "#+BEGIN_SRC <language>"`